src/components/screens/ChapterScreen/index.js

```diff
@@ -118,6 +119,12 @@ const Chapter = ({
       <Content>
         <Title>{title}</Title>
         <Description>{description}</Description>
+        {!tutorialUpToDate && (
+          <div className="translation-aside">
+            {fetchStatusUpdate.guidenotupdated}{' '}
+            <a href={githubFileUrl}>{fetchStatusUpdate.pullrequestmessage}</a> .
```

Comment:
Use the `Link` component instead of an `a`

Reply:
@domyen with the latest commit the anchor tag was replaced with a Link component.